.golangci.yml

```diff
@@ -49,6 +50,11 @@ linters-settings:
     line-length: 170
     tab-width: 4
 
+  staticcheck:
+    checks:
+      - all
+      - "-SA1019"
```

Comment:
That's actually good rule and we should address highlighted problems.

Reply:
> That's actually good rule and we should address highlighted problems.

I also agree it's a good rule, if it wasn't for two facts:
1/ we deprecate things and we never fix the linter warnings that appear as a result of deprecation; I even [asked the author](https://github.com/percona/pmm/issues/1541#issuecomment-1436500268) to address them once, but they are still there :)
2/ deprecation warnings are so numerous that they make it super challenging to find what you need in the logs (pls see attachement)
[14_Run linters.txt](https://github.com/percona/pmm/files/11372274/14_Run.linters.txt)

I suggest we temporarily disable this rule until we fix all deprecation warnings.

I believe we need to hear more opinions.  @BupycHuk @michal-kralik @PavelKhripkov 
What do you think?

Reply:
If we will keep this linter disabled we need ticket to enable it back. Otherwise we will forget about it.

Reply:
> If we will keep this linter disabled we need ticket to enable it back. Otherwise we will forget about it.

Sure, I see no problem :)

Reply:
I agree with Artem that we should keep this linter even if we ignore it for now.

Reply:
> I agree with Artem that we should keep this linter even if we ignore it for now.

So, what do we do now? Disable, create a ticket for re-enabling it and then address the linter warnings?

Reply:
No, we shouldn't disable it, instead, we should address warnings.

Reply:
@JiriCtvrtka already fixed some of them, and @cfsalguero is supposed to fix the ones related to DBaaS. So we are fixing them.